assets/css/style.css

```diff
@@ -633,3 +633,118 @@ small {
     grid-template-columns: 40% 1fr;
   }
 }
+/* My CSS Code */
+
+/* #1 id selector for form background */
+#contact-form{
+  background: white;
+  padding: 1rem;
+}
+/* #2 container of the form to center it  */
+.container-form{
+  height: 100%;
+  margin: auto;
+}
+
+/* #3 the title in top of the form  */
+.title-form-top{
+  color: var(--text-gray);
+  font-size: 1.8rem;
+  text-align: justify;
+  text-justify: auto;
+  margin-bottom: 30px;
+}
+
+/* #4 editing the standard input to remove borders */
+input{
```

Comment:
I see you're using `border: none!important;` , but the use of `!important` could make future styling more difficult. Is there a specific reason for using `!important` here?